packages/actor-rdf-resolve-quad-pattern-federated/package.json

```diff
@@ -62,7 +62,10 @@
       "ts",
       "js"
     ],
-    "collectCoverage": true
+    "collectCoverage": true,
+    "testPathIgnorePatterns": [
```

Comment:
This should not be necessary.

Reply:
If we keep the mocked RDF Metadata Aggregate mediator in a separate file (cfr. [this comment](https://github.com/comunica/comunica/pull/821#discussion_r692218718)), the mock-folder is likely to remain in the ignored testpaths?